src/components/workbench/Input.ts

```diff
@@ -1,5 +1,6 @@
 import { InputBox, QuickOpenBox, QuickPickItem } from "vscode-extension-tester";
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
this is not used here but it's interface so there's possibility that some of projects is using this so i guess we should keep this

Reply:
we can skip that for now